### PR TITLE
[GEN-1170] Add missing profile to data guide

### PR DESCRIPTION
**Pull request by rxu17**
**Purpose:** Adds the missing center profile for our GOLD center. The docker image tagged at `sagebionetworks/genie-data-guide:latest` on dockerhub needs to be rebuilt in order for this to work on Nextflow Workflow Tower.

I don't have the permissions to push to `sagebionetworks` on dockerhub, but tested `consortium_release` on `nf-genie` on the rebuilt image locally and it works (data guide on test pipeline gets outputted): https://www.synapse.org/#!Synapse:syn21895009 

Once this gets pushed to main, the docker image will rebuild on dockerhub, and the nextflow workflow tower run will work

### CODE REVIEW

**Comment by rxu17** [comment]
@thomasyu888 Could you add me as admin or add `@Sage-Bionetworks/genie-reviewers` to have write access to the repo?

**Review by thomasyu888 — APPROVED** [review]
🔥 LGTM!  Just a comment, but going to pre-approve.  Ill leave it to you to decide whether you want to push this when processing is still happening

**Comment by rxu17** [comment]
> 🔥 LGTM! Just a comment, but going to pre-approve. Ill leave it to you to decide whether you want to push this when processing is still happening

Good point, I'll wait for now.

**Pull request merged**
